scripts/Expand-Exchange.ps1

```diff
@@ -0,0 +1,26 @@
+[CmdletBinding()]
+param(
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $ExchangeServerVersion
+
+)
+
+try {
+    Start-Transcript -Path C:\cfn\log\Expand-Exchange.ps1.txt -Append
+    $ErrorActionPreference = "Stop"
+    
+    if($ExchangeServerVersion -eq "2013") {
+        Invoke-Command -ScriptBlock {Start-Process cmd.exe '/c c:\Exchangeinstall\Exchange2013-x64-cu16.exe /extract:c:\Exchangeinstall /passive' -NoNewWindow -Wait} 
+    }
+    elseif ($ExchangeServerVersion -eq "2016") {
+        $isoPath = 'C:\Exchangeinstall\ExchangeServer2016-x64-cu5.iso'
```

Comment:
Yeah, might be best to create a Mapping in the CFN template that has the different versions of Exchange with their respective download URLs.